sw/device/lib/testing/json/mem.c

```diff
@@ -0,0 +1,62 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#define UJSON_SERDE_IMPL 1
+#include "sw/device/lib/testing/json/mem.h"
+
+#include "sw/device/lib/base/memory.h"
+#include "sw/device/lib/testing/test_framework/ujson_ottf.h"
+
+#define MODULE_ID MAKE_MODULE_ID('j', 's', 'm')
+
+status_t ujcmd_mem_read32(ujson_t *uj) {
+  mem_read32_req_t op;
+  mem_read32_resp_t resp;
+  uintptr_t address;
```

Comment:
Do we enforce C89 style declaration? I don't see much value of upfront declaration.

Also why `address` instead of using `op.address` directly?

Reply:
Whether C89 style provides value is altogether unimportant, and I'd appreciate it if you avoided trying to micromanage how people write code. (Though I do need help with Rust: I'm a total newbie there! :joy: )

That said, I didn't intend to leave the separate variable (was futzing around and it was leftovers). Changed.